core/src/main/java/com/novoda/noplayer/exoplayer/MediaSourceFactory.java

```diff
@@ -36,6 +38,10 @@ MediaSource create(ContentType contentType,
                         handler,
                         eventListener
                 );
+            case DASH:
+                return new DashMediaSource(uri, mediaDataSourceFactory,
```

Comment:
We need to have DASH enabled in order to have multiple audio sources in a `media source`.